README.md

```diff
@@ -1,87 +1,81 @@
-# wavefront-lambda-go [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go)
+# wavefront-lambda-go
 
-This is a Wavefront Go wrapper for AWS Lambda to enable reporting standard lambda metrics and custom app metrics directly to wavefront.
+[![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-lambda-go)
+[![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-lambda-go)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-lambda-go)
 
-## Requirements
-Go 1.x
+A Go wrapper for AWS Lambda so you can monitor everything from your [Wavefront](https://wavefront.com) dashboard
 
 ## Installation
-```
+
+Using `go get`
+
+```bash
 go get github.com/wavefronthq/wavefront-lambda-go
 ```
 
-## Environment variables
-WAVEFRONT_URL = https://\<INSTANCE>.wavefront.com  
-WAVEFRONT_API_TOKEN = Wavefront API token with Direct Data Ingestion permission.  
-REPORT_STANDARD_METRICS = Set to False or false to not report standard lambda metrics directly to wavefront.  
-
 ## Usage
 
-Wrap your AWS Lambda handler function with wavefront_lambda.Wrapper(LambdaHandler).
+To connect your Lambda functions to Wavefront, you'll need to set two environment variables, import this module, and wrap your AWS Lambda handler function with `wavefront_lambda.Wrapper(LambdaHandler)`. The environment variables you'll need to set are:
+
+* `WAVEFRONT_URL`: The URL of your Wavefront instance (like, `https://myinstance.wavefront.com`).
+* `WAVEFRONT_API_TOKEN`: Your Wavefront API token (see the [docs](https://docs.wavefront.com/wavefront_api.html) how to create an API token).
```

Comment:
Need to still mention REPORT_STANDARD_METRICS as an environment variable. Its optional but needs a mention in this doc.

Reply:
It is mentioned, just a little lower in the doc. The idea of mentioning it later in the README (line 55) Is that it ties specifically to the standard metrics that the module sends to Wavefront. Because of that I thought it made sense to keep those two things together. I could add it here too and mention it’s discussed in more detail later on. Does that make sense?

Reply:
oh ok thats makes sense, I also didn't read through the entire PR before commenting. Thanks